packages/vue-components/src/panels/PanelBase.js

```diff
@@ -180,15 +233,30 @@ export default {
         this.src = this.src.split('#')[0];
       }
     }
+
     // Edge case where user might want non-expandable card that isn't expanded by default
     const notExpandableNoExpand = !this.expandableBool && this.expanded !== 'false';
 
     // Set local data to computed prop value
 
     // Ensure this expr ordering is maintained
     this.localExpanded = notExpandableNoExpand || this.expandedBool;
-    // If it is expanded, load the retriever immediately.
```

Comment:
could preserve these in favour of `if (this.minimizedBool && this.localExpanded) { ... }` and `this.wasRetrieverLoaded = this.localExpanded;` in `mounted()`

Reply:
Sorry, for the preserving, can I check which one are u referring to? :O

Reply:
Can I check if you mean: 
1. remove `if (this.minimizedBool && this.localExpanded) { ... }` and `this.wasRetrieverLoaded = this.localExpanded;` from mounted()
2. keep `this.wasRetrieverLoaded` in created()

Reply:
both, also we could keep the line below `this.wasRetrieverLoaded = this.localExpanded` which is:
`this.localMinimized = this.minimizedBool`

note you have
```
if (this.minimizedBool && this.localExpanded) {
  this.localMinimized = true
```
in `created()`

and then 

```
if (this.minimizedBool && !this.localExpanded) {
  // If panel is minimized, simply close the panel.
  this.close();
  return;
}
```
in `mounted()` (where `close()` also sets `localMinimized = true`)

Reply:
Ah, I think I see what you mean! Those two branches are actually redundant. 

I will make the changes accordingly. Do let me know if I happen to misinterpret your explanation.